app/boards/shields/nice_view/README.md

```diff
@@ -1,5 +1,15 @@
 # nice!view
 
-The nice!view is a low power, high refresh rate display meant to replace I2C OLEDs traditionally used.
+The nice!view is a low-power, high refresh rate display meant to replace I2C OLEDs traditionally used.
 
-This shield requires that an `&nice_view_spi` labelled SPI bus is provided with _at least_ MOSI, SCK, and CS pins defined.
+This shield requires that an `&nice_view_spi` labeled SPI bus is provided with _at least_ MOSI, SCK, and CS pins defined.
+
+## Disable custom widget
+
+The nice!view shield includes a custom vertical widget. To use the built-in ZMK one, add the following items to your `.conf` file:
+
+```
+CONFIG_ZMK_DISPLAY_STATUS_SCREEN_BUILT_IN=y
+CONFIG_ZMK_LV_FONT_DEFAULT_SMALL_MONTSERRAT_26=y
+CONFIG_LV_FONT_DEFAULT_MONTSERRAT_26=y
```

Comment:
Can these two settings be selected for the built-in screen in Kconfig instead?

Reply:
I tried adding them as a default choice under the built-in screen, but that didn't work. You can't select choices directly, so not sure how to "force" a choice.

Reply:
I did some experimenting and this compiles since we use named `choice`s:
```diff
diff --git a/app/boards/shields/nice_view/Kconfig.defconfig b/app/boards/shields/nice_view/Kconfig.defconfig
index 2656a2cd..323eafe6 100644
--- a/app/boards/shields/nice_view/Kconfig.defconfig
+++ b/app/boards/shields/nice_view/Kconfig.defconfig
@@ -53,7 +53,21 @@ config NICE_VIEW_WIDGET_INVERTED
 
 endif # ZMK_DISPLAY_STATUS_SCREEN_CUSTOM
 
-config ZMK_DISPLAY_STATUS_SCREEN_BUILT_IN
-    select LV_FONT_MONTSERRAT_26
+if ZMK_DISPLAY_STATUS_SCREEN_BUILT_IN
+
+config LV_FONT_MONTSERRAT_26
+    default y
+
+choice ZMK_LV_FONT_DEFAULT_SMALL
+    default ZMK_LV_FONT_DEFAULT_SMALL_MONTSERRAT_26
+
+endchoice
+
+choice LV_FONT_DEFAULT
+    default LV_FONT_DEFAULT_MONTSERRAT_26
+
+endchoice
+
+endif
 
 endif # SHIELD_NICE_VIEW
```
But it still doesn't set the right choice, so I guess [this comment](https://github.com/zephyrproject-rtos/zephyr/issues/6948#issuecomment-379210116) still applies:
>  I just looked into defaults on choices btw, and the C tools in Zephyr haven't been patched to prefer later defaults there either, so it's a bit inconsistent: Later defaults override earlier defaults for symbols, but earlier defaults override later defaults for choices.
>
> That means that the "base" choice definition (where the symbols are defined) can't have any defaults (with satisfied conditions). Otherwise, those will take precedence over the defaults in the Kconfig.* files, which are included later.

Reply:
Yep, I tried exactly this with the same results. Agreed that right now this isn't possible.